internal/archive/archive_test.go

```diff
@@ -335,22 +335,67 @@ func (s *httpSuite) TestArchiveLabels(c *C) {
 	}
 
 	_, err = archive.Open(&options)
-	c.Assert(err, IsNil)
+	c.Assert(err, ErrorMatches, `.*\bno Ubuntu section`)
```

Comment:
Can we make the test above into a table-test and add the specific error when we expect one, no wildcards please.

And in fact, this test is about testing the labels and this new case is setting none, why is that?

Reply:
Removed the wildcards. The last test is basically testing error case on an invalid unknown label.

Let me know if you still think table test would be better here.